library/couchbase

```diff
@@ -10,13 +10,60 @@ GitCommit: 705652faaef8f56988ba13d181cb00109c7263d4
 Directory: community/couchbase-server/7.0.0-beta
 
 Tags: latest, enterprise, 6.6.2, enterprise-6.6.2
-GitCommit: 9258627737f48d0b7c1015f48fdfd6242aefba6c
+GitCommit: f8bc62b9fc66a732b45a9a57fc7495d9940d10b7
 Directory: enterprise/couchbase-server/6.6.2
 
 Tags: community, community-6.6.0
-GitCommit: 78cbcaa2c90ce4c975299e7cbfdce146a7bab081
+GitCommit: f8bc62b9fc66a732b45a9a57fc7495d9940d10b7
 Directory: community/couchbase-server/6.6.0
 
 Tags: 6.0.5, enterprise-6.0.5
-GitCommit: fdd43ff22dbb4b10f262e95acef6b945dfacf214
+GitCommit: f8bc62b9fc66a732b45a9a57fc7495d9940d10b7
 Directory: enterprise/couchbase-server/6.0.5
+
+# One-off updates of older versions as we have updated the
+# Ubuntu base image version to address security vulnerabilities
+
+Tags: 6.6.1, enterprise-6.6.1
+GitCommit: f8bc62b9fc66a732b45a9a57fc7495d9940d10b7
+Directory: enterprise/couchbase-server/6.6.1
+
+Tags: 6.6.0, enterprise-6.6.0
+GitCommit: f8bc62b9fc66a732b45a9a57fc7495d9940d10b7
+Directory: enterprise/couchbase-server/6.6.0
+
+Tags: 6.5.1, enterprise-6.5.1
+GitCommit: f8bc62b9fc66a732b45a9a57fc7495d9940d10b7
+Directory: enterprise/couchbase-server/6.5.1
+
+Tags: 6.5.0, enterprise-6.5.0
+GitCommit: f8bc62b9fc66a732b45a9a57fc7495d9940d10b7
+Directory: enterprise/couchbase-server/6.5.0
+
+Tags: 6.0.4, enterprise-6.0.4
+GitCommit: f8bc62b9fc66a732b45a9a57fc7495d9940d10b7
+Directory: enterprise/couchbase-server/6.0.4
+
+Tags: 6.0.3, enterprise-6.0.3
+GitCommit: f8bc62b9fc66a732b45a9a57fc7495d9940d10b7
+Directory: enterprise/couchbase-server/6.0.3
+
+Tags: 6.0.2, enterprise-6.0.2
+GitCommit: f8bc62b9fc66a732b45a9a57fc7495d9940d10b7
+Directory: enterprise/couchbase-server/6.0.2
+
+Tags: 6.0.1, enterprise-6.0.1
+GitCommit: f8bc62b9fc66a732b45a9a57fc7495d9940d10b7
+Directory: enterprise/couchbase-server/6.0.1
+
+Tags: community-6.6.0
+GitCommit: f8bc62b9fc66a732b45a9a57fc7495d9940d10b7
+Directory: community/couchbase-server/6.6.0
+
```

Comment:
Ah, this is still a duplicate (enterprise is at 6.6.2 but community appears to still be on 6.6.0):

```suggestion

```